cmd/notation/verify.go

```diff
@@ -246,6 +246,7 @@ func getVerifier(ctx context.Context) (notation.Verifier, error) {
 	if err != nil {
 		return nil, err
 	}
+	crlFetcher.DiscardCacheError = true // discard cache error
```

Comment:
Should we throw warning when we are discarding cache error? IMO user would like to know if there caching mechanism is not working

Reply:
Added log.

Reply:
We should not always log `Discard any crl cache error`. We should only log when there is an error and discarded.

Reply:
Then this log needs to be added in notation-core-go, because from notation-go level the error has already been discarded. Removed from notation CLI.

Reply:
I think for notation-core-go, we cannot use the logger. Should we add the warning log in notation-go?

Reply:
Can we please open issue to track this issue.

Reply:
Issue created: https://github.com/notaryproject/notation-core-go/issues/242